src/components/block/index.ts

```diff
@@ -690,11 +685,47 @@ export default class Block {
         pluginsContent = this.toolInstance.render();
 
     contentNode.appendChild(pluginsContent);
-    wrapper.appendChild(contentNode);
+
+    let wrappedContentNode: HTMLElement = contentNode;
+
+    [...this.tunesInstances.values(), ...this.defaultTunesInstances.values()]
+      .forEach((tune) => {
+        if (_.isFunction(tune.wrap)) {
+          wrappedContentNode = tune.wrap(wrappedContentNode);
+        }
+      });
+
+    wrapper.appendChild(wrappedContentNode);
 
     return wrapper;
   }
 
+  /**
+   * Instantiate Block Tunes
+   *
+   * @param tunesData - current Block tunes data
+   * @private
+   */
+  private composeTunes(tunesData: {[name: string]: BlockTuneData}): void {
+    [
+      ...this.tunes.externalTools.values(),
+      ...this.tunes.internalTools.values(),
+    ].forEach((tune) => {
+      const collection = tune.isInternal ? this.defaultTunesInstances : this.tunesInstances;
```

Comment:
What's the reason for separation for `this.defaultTunesInstances` and `this.tunesInstances`? As I can see, all the usages are merging them back.

Reply:
Tune instance doesn't provide any flag if it's internal or not, so we need to split them to know how to render in block settings